ts/src/xt.ts

```diff
@@ -4603,7 +4603,7 @@ export default class xt extends Exchange {
             const defaultRecvWindow = this.safeString (this.options, 'recvWindow');
             const recvWindow = this.safeString (params, 'recvWindow', defaultRecvWindow);
             const timestamp = this.numberToString (this.nonce ());
-            body = params;
+            body = this.extend (params);
```

Comment:
@ttodua let’s only extend it if we need to, do not “pre-extend” it here, move it under the if-clause cause that is still cheaper than extending it all the time